bundles/org.openhab.binding.dreamscreen/src/main/resources/ESH-INF/config/config.xml

```diff
@@ -10,10 +10,10 @@
 			<description>The serial number of the DreamScreen device</description>
 		</parameter>
 
-        <parameter name="groupId" type="text">
-            <label>Group Id</label>
-            <description>Logical group Id to control multiple devices at once</description>
-        </parameter>
+		<parameter name="groupId" type="text">
```

Comment:
please run "mvn spotless:apply" to fix formatting, this is required to pass the review